src/utils/index.js

```diff
@@ -97,3 +97,7 @@ function formatDateTime (isoDate) { // eslint-disable-line no-unused-vars
     // Throw Error, iso-timedate cannot be parsed into valid format
   } else throw new Error('Iso timedate is unable to be parsed.', isoDate)
 }
+
+export function getCommunityUrl () {
+  return window.location.protocol + '//' + window.location.host + '/community/'
+}
```

Comment:
Has this approach been tested yet?

Reply:
I've tested it in that it produces the expected url in the browser. So if community app is on `{host}/community/` then this will work. But I doubt it's been tested in HPOS with community app running.

Reply:
I will be able to test it later this week.